integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/datamap/CarbonDropDataMapCommand.scala

```diff
@@ -116,8 +116,12 @@ case class CarbonDropDataMapCommand(
         // drop index datamap on the main table
```

Comment:
please modify comment if this branch is for dropping MV datamap also

Reply:
fixed